index.test.js

```diff
@@ -527,6 +527,50 @@ describe('Configure AWS Credentials', () => {
         })
     });
 
+    test('role assumption session token provided, no role assumption duration provided', async () => {
```

Comment:
For both these tests can you modify the title to be more descriptive of what is being tested?

e.g. `test('sets durationSeconds to one hour when session token provided and no duration is provided')` and `test('sets durationSeconds to one 6 hours no session token or duration is provided')`